pkg/lib/config/bot_protection_env.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+type BotProtectionEnvironmentConfig struct {
+	CloudflareEndpoint  string `envconfig:"BOT_PROTECTION_CLOUDFLARE_ENDPOINT" default:"https://challenges.cloudflare.com/turnstile/v0/siteverify"`
```

Comment:
First of all, do not put the real value here. Keep the real value in the original place. If the envvar is set, then use the envvar, otherwise, keep using the real value.